### PR TITLE
[12.0] [IMP] Payment Return: usability

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
* New total field on payment returns
* Total footers
* Display the bank account from the imported file on the payment orders (useful when using transfer accounts)

### CODE REVIEW

**Review by chrisdec — APPROVED** [review]
Functional Test

File: account_payment_return/views/payment_return_view.xml [hunk 1]
```diff
@@ -43,8 +44,8 @@
                                                     ('account_id.internal_type', '=', 'receivable'),
                                                     ('reconciled', '=', True)]"
                                             />
-                                    <field name="amount"/>
-                                    <field name="expense_amount" />
+                                    <field name="amount" sum="Total amount"/>
+                                    <field name="expense_amount" sum="Total expense amount"/>
```
Comment (sbidoul) [inline account_payment_return/views/payment_return_view.xml]: The string for `expense_amount` is `Charges Amount`, so it's better to use the same vocabulary for the total.

**Review by sbidoul — CHANGES_REQUESTED** [review]
A small vocabulary issue, otherwise looks good (code review).

File: account_payment_return/views/payment_return_view.xml (hunk 1 above)
Comment (ThomasBinsfeld) [reply account_payment_return/views/payment_return_view.xml]: Done, thanks!

**Review by sbidoul — APPROVED** [review]

**Pull request merged**